src/libCommon/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
+
+find_package(OpenCV REQUIRED)
```

Comment:
not needed here, `find_package(OpenCV REQUIRED)` is already called in top-level CMakeLists, so `OpenCV_INCLUDE_DIRS` and `OpenCV_LIBS` are already defined